resources/config/laravel-link-checker.php

```diff
@@ -41,6 +41,12 @@
              * The to address to be used by the mail reporter.
              */
             'to_address' => '',
+
+            /**
+             * The to address to be used by the mail reporter.
```

Comment:
Well - My first PR and I've embarrassed myself. Not only am I unsure if this can work (since I can't seem to work out how to get this test on homestead), I've also missed obvious stuff such as this! Sorry to add to your workload Freek.